Wave/Wave/SingleSelectionSetting.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  SingleSelectionSetting.swift
+//  Wave
+//
+//  Created by Jimmy McDermott on 7/5/16.
+//  Copyright © 2016 BlueLine Labs. All rights reserved.
+//
+
+import Foundation
+
+public final class SingleSelectionSection: SectionType, SettingType {
+    
+    public var possibleValues: [Any]
+    public var enableCustom = false
+    public var customOption: RowType?
+    public var name: String
+    private var settings = [StringSetting]()
+    
+    public init(possibleValues: [Any], enableCustom: Bool, customOption: RowType?, name: String) {
```

Comment:
How come this is `[Any]` instead of `[String]`? I guess it could work, as long as whatever is being passed in can be represented as a string. If this was your train of thought, it should probably still be `AnyObject` so it can be serialized to a plist.